utilities/after_success.sh

```diff
@@ -12,8 +12,20 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" ]; then
     if [ "${TRAVIS_PULL_REQUEST}" == "false" -a "${TRAVIS_BRANCH}" == "master" ]; then
         source ./utilities/integration_test_env.sh
         SITE_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -Ev '(^\[|\w+:)')
+        echo "Used the maven-help-plugin to determine that the version is $SITE_VERSION"
+        if [ "$SITE_VERSION" == "" ]; then
+            echo "Could not determine the version, so we're exiting."
+            exit 1
+        fi
         if [ "${SITE_VERSION##*-}" != "SNAPSHOT" ]; then
-            # Deploy site if not a SNAPSHOT
+            # Deploy Maven artifacts (if they don't exist yet) and update artifact version in READMEs.
+            ARTIFACT_EXISTS=$(mvn wagon:exist -Dwagon.url=https://oss.sonatype.org/content/repositories/releases/com/google/gcloud/gcloud-java/$SITE_VERSION)
+            if [[ "$ARTIFACT_EXISTS" == *"does not exists."* -a "$ARTIFACT_EXISTS" != "" ]]; then
```

Comment:
Is this line actually working? If I remember correctly `-a` does not work with `[[` (I might be wrong). Something like the following should work:
```bash
if [[ "$ARTIFACT_EXISTS" == *"does not exists."* &&  "$ARTIFACT_EXISTS" != "" ]]; then
```
Also is the second check `"$ARTIFACT_EXISTS" != ""` ever used?